### ISSUE TITLE
Accuracy problem

### ISSUE THREAD

**Issue by wyq0227** [clcarwin/sphereface_pytorch#31]
I used my own train.py to train the model which has same data processing and same parameters.
But my test result is 98.38% using clcarwin's lfw_eval.py in Python2.7 and Pytorch0.4.0.
I don't know why my result is lower than clcarwin's.
Does anybody can solve my problem?Thanks.

**Comment by Gerkam** [comment]
@wyq0227 How many epochs and iterations have you done?

**Comment by wyq0227** [comment]
@Gerkam 20, the same as clcarwin's.
I found that the problem was that I didn't change the images' channels. I read them in RGB in training process but I read them in BGR in testing process. So there is a little deviation in accuracy.
